scripts/crud

```diff
@@ -0,0 +1,215 @@
+    required.add_argument("-u", "--uuid",  type=str, default=None, help="uuid", required=True)
+    parser.add_argument("-s", "--save", type=str, default=None, help="save to file")
+
+    subparsers = parser.add_subparsers()
+
+    # Create
+    create_parser = subparsers.add_parser('create', help="Create k/v")
+    create_group = create_parser.add_argument_group('required arguments')
+    create_group.add_argument("-k", "--key",   type=str, default=None, help="key", required=True)
+    group = create_group.add_mutually_exclusive_group()
+    group.add_argument("-v", "--value", type=str, default="", help="value")
+    group.add_argument("-f", "--file",  type=str, default=None, help="file to upload")
+    create_parser.set_defaults(func=create_handler)
+
+    # Read
+    read_parser = subparsers.add_parser('read', help="Read k/v")
+    read_group = read_parser.add_argument_group('required arguments')
+    read_group.add_argument("-k", "--key",   type=str, default=None, help="key", required=True)
+    read_parser.set_defaults(func=read_handler)
+
+    # Update
+    update_parser = subparsers.add_parser('update', help="Update k/v")
+    update_group = update_parser.add_argument_group('required arguments')
+    update_group.add_argument("-k", "--key",   type=str, default=None, help="key", required=True)
+    update = update_group.add_mutually_exclusive_group()
+    update.add_argument("-v", "--value", type=str, default=None, help="value")
+    update.add_argument("-f", "--file",  type=str, default=None,  help="file to upload")
+    update_parser.set_defaults(func=update_handler)
+
+    # delete
+    delete_parser = subparsers.add_parser('delete', help="Delete k/v")
+    delete_group = delete_parser.add_argument_group('required arguments')
+    delete_group.add_argument("-k", "--key",   type=str, default=None, help="key", required=True)
+    delete_parser.set_defaults(func=delete_handler)
+
+    # has
+    has_parser = subparsers.add_parser('has', help="Determine whether a key exists within a DB by UUID")
+    has_group = has_parser.add_argument_group('requred arguments')
+    has_group.add_argument("-k", "--key",   type=str, default=None, help="key", required=True)
+    has_parser.set_defaults(func=delete_handler)
+
+    # keys
+    keys_parser = subparsers.add_parser('keys', help="Get all keys for a DB by UUID")
+    keys_parser.set_defaults(func=keys_handler)
+
+    # size
+    size_parser = subparsers.add_parser('size', help="Determine the size of the DB by UUID")
+    size_parser.set_defaults(func=size_handler)
+
+    args = parser.parse_args()
+
+    try:
+        # if user gave us a file we need to read it as our value
+        args.value = open(args.file).read()
+    except IOError as e:
+        print("failed to read value from file: {}".format(e.strerror))
+        exit(1)
+
+
+
+
+    resp = args.func(args)
+
+    # save to a file...
+    if args.save:
```

Comment:
You've changed how this works. Save is only applicable to a read operation. It was intended to download a "value" as a file and not the entire response.